_jobs/2018-08-06-openfisca-lean-ux.md

```diff
@@ -0,0 +1,74 @@
+---
+roles: un·e Lean UX
+startup: openfisca
+techno:
+open: true
+---
+
+[OpenFisca](https://openfisca.org/fr/) cherche un·e [Lean UX](https://svpg.com/the-product-designer-role/) pour un CDD (6 mois).
+
+Souhaitez-vous aider des gouvernements et la société civile à construire des services innovants, qui aident les citoyens à mieux comprendre le système socio-fiscal ?
+
+Souhaitez-vous accompagner le Parlement dans la simulation de l'impact des lois avant de les voter ?
+
+Souhaitez-vous participer au déploiement en France et à l'international d'un logiciel libre avec une communauté de contributeurs qui grandit ?
+
+<!--more-->
+
+## Qui sommes nous ?
+
+Vous rejoindrez l'[équipe autonome](https://beta.gouv.fr/2016/11/28/equipes-autonomes) qui développe OpenFisca.
+
+[OpenFisca](https://openfisca.org/fr/) a pour mission de transformer le code législatif en code logiciel, puis le code logiciel en code législatif.
+
+Entièrement libre et gratuit, modulable et réutilisable, OpenFisca est un moteur de microsimulation des systèmes socio-fiscaux.
+
+OpenFisca permet de comprendre les lois, d'analyser leurs changements et de simuler des situations, ayant ainsi pour but d’aider les citoyens à mieux comprendre les réformes qui impactent leur vie quotidienne.
+
+## Le rôle
+
+Nous comprenons le rôle de Lean UX comme quelqu'un·e qui :
+
+- [pense que l’UX est une façon de résoudre les problèmes des usagers, plutôt que de développer la prochaine fonctionnalité dernier cri](https://www.smashingmagazine.com/2014/01/lean-ux-manifesto-principle-driven-design/#measuring-kpis-over-undefined-success-metrics) ;
```

Comment:
Je pense que ce serait interessant de décrire le genre de chose que la personne a fait dans le passé.
exemple : 
Vous avez été product manager, lead designer d'une API ou travaillé sur l'experience développeur d'un produit.

Reply:
OK je vais itérer sur ce sujet.

Reply:
Je vais ajouter une phrase dans la partie `Votre profil`